### PR TITLE
Apps section minus content types 

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

File: admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/global-messages_it.properties [hunk 1]
```diff
@@ -8,7 +8,7 @@ jacms.menu.contentAdmin=Contenuti
 jacms.menu.contentAdmin.new=Nuovo Contenuto
 jacms.menu.contentAdmin.new.long=Per inserire nuovi Contenuti nell''archivio dei Contenuti  (o "Lista Contenuti") del portale
 jacms.menu.contentAdmin.list=Lista Contenuti
-jacms.menu.contentAdmin.list.long=Per gestire i Contenuti: � possibile modificarli, approvarli, sospenderli, rimuoverli
+jacms.menu.contentAdmin.list.long=Per gestire i Contenuti: \u00ef\u00bf\u00bd possibile modificarli, approvarli, sospenderli, rimuoverli
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/global-messages_it.properties]: shouldn't this just be \u00e8 ?

File: admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/global-messages_it.properties [hunk 2]
```diff
@@ -54,20 +57,20 @@ jacms.menuSection.longDesc=java Agile CMS - Contenuti
 jacms.shortcut.content.new.shortDesc=Nuovo Contenuto
 jacms.shortcut.content.new.longDesc=Per inserire nuovi Contenuti nell''archivio dei Contenuti  (o "Lista Contenuti") del portale
 jacms.shortcut.content.list.shortDesc=Lista Contenuti
-jacms.shortcut.content.list.longDesc=Per gestire i Contenuti: � possibile modificarli, approvarli, sospenderli, rimuoverli
+jacms.shortcut.content.list.longDesc=Per gestire i Contenuti: \u00ef\u00bf\u00bd possibile modificarli, approvarli, sospenderli, rimuoverli
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/global-messages_it.properties]: shouldn't this just be \u00e8 ? 
same applies to the rest of the file

File: admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/package_it.properties [hunk 3]
```diff
@@ -91,14 +120,14 @@ note.button.approve=Approva il Contenuto rendendolo disponibile al pubblico
...
 note.button.delete=Elimina il Contenuto dall''archivio
 note.button.previewContent=Mostra in anteprima il Contenuto come fosse pubblicato nella Pagina scelta, senza effettivamente pubblicarlo
-note.button.addGroups=Aggiungi uno o pi� gruppi ai contenuti selezionati
-note.button.removeGroups=Rimuovi uno o pi� gruppi ai contenuti selezionati
-note.button.addCategories=Aggiungi una o pi� categorie ai contenuti selezionati
-note.button.removeCategories=Rimuovi una o pi� categorie ai contenuti selezionati
+note.button.addGroups=Aggiungi uno o pi\u00ef\u00bf\u00bd gruppi ai contenuti selezionati
+note.button.removeGroups=Rimuovi uno o pi\u00ef\u00bf\u00bd gruppi ai contenuti selezionati
+note.button.addCategories=Aggiungi una o pi\u00ef\u00bf\u00bd categorie ai contenuti selezionati
+note.button.removeCategories=Rimuovi una o pi\u00ef\u00bf\u00bd categorie ai contenuti selezionati
```
Comment (nicpuddu) [inline admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/package_it.properties]: shouldn't these be just \u00f9

File: admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/entity/include/entity-type-list.jsp [hunk 4]
```diff
@@ -49,49 +50,50 @@
 </s:if>
 <s:else>
     <ol class="breadcrumb page-tabs-header breadcrumb-position">
-        <li class="page-title-container"><s:text
-                name="%{'title.' + entityManagerName + '.management'}"/></li>
+        <li>
+            <s:text name="breadcrumb.app" /></li>
+        <li>
+            <s:text name="breadcrumb.jacms"/>
+        </li>
```
Comment (paddeo) [inline admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/entity/include/entity-type-list.jsp]: we can not put here specific property keys. please, use the pattern `name="%{'_something_.' + entityManagerName + '._somethingelse_'}"`

**Comment by nicpuddu** [comment]
🚢 🇮🇹

**Pull request merged**